libs/IO/Image.h

```diff
@@ -107,6 +109,49 @@ class IO_API CImage
 	#endif
 
 protected:
+    template<typename T>
```

Comment:
this is a useful function in general, pls move it in Util.h and call it ComputePrecentileMinMax

Reply:
Done :+1: